packages/x-data-grid/src/hooks/features/virtualization/useGridVirtualScroller.tsx

```diff
@@ -583,7 +583,6 @@ export const useGridVirtualScroller = () => {
     }),
     getScrollerProps: () => ({
       ref: scrollerRef,
-      tabIndex: -1,
```

Comment:
Without the explicit tabIndex here, Firefox focuses on the virtual scroller element:

### Chromium:
https://github.com/user-attachments/assets/3b0444da-ecf7-4619-bdf5-166744c471cb

### Firefox:
https://github.com/user-attachments/assets/406ab2a9-78c8-4832-aa52-8c168711128c

Reply:
Does the Scrollbar may also need `tabIndex: -1` to be removed?

![Screenshot from 2024-07-19 10-07-35](https://github.com/user-attachments/assets/09197942-dbe9-4a44-a774-6ca4313fa488)

Reply:
I'll look into the scrollbar.

@cherniavskii Wouldn't that be a firefox issue? That element is not listening to any key event, only to wheel/scroll ones. I could restore the tabIndex only for FF and we file a bug on their tracker?

Reply:
I've added `aria-hidden=true` for the scrollbar, same as ag-grid does, it's probably the best aria tag for that case.

@cherniavskii For the Firefox issue, wouldn't that be a Firefox bug? Can you test on Safari to see what's the behavior? If Chrome & Safari agree and Firefox disagrees, we could add the tabindex only as a workaround for FF and file an issue with them.
Looks vaguely similar to this: https://bugzilla.mozilla.org/show_bug.cgi?id=1069739

Reply:
I checked Chrome, Safari, and Firefox, and the issue is only reproducible in Firefox.

> we could add the tabindex only as a workaround for FF and file an issue with them

Yeah, that makes sense 👍🏻

> Looks vaguely similar to this: [bugzilla.mozilla.org/show_bug.cgi?id=1069739](https://bugzilla.mozilla.org/show_bug.cgi?id=1069739)

<img width="138" src="https://github.com/user-attachments/assets/4508cf03-a034-4849-9843-c36be86a5805"> 😅

Reply:
Pretty common lifespan for a firefox bug, they've been around for longer and have way less resources :|